dandi/organize.py

```diff
@@ -307,9 +308,28 @@ def _assign_obj_id(metadata, non_unique):
     # Avoid heavy import by importing within function:
     from .pynwb_utils import get_object_id
 
-    msg = "%d out of %d paths are not unique" % (len(non_unique), len(metadata))
+    msg = "%d out of %d paths are not unique." % (len(non_unique), len(metadata))
```

Comment:
Use f-strings here.

Reply:
yeah, should be consistent.  
```suggestion
    msg = f"{len(non_unique)} out of {len(metadata)} paths are not unique."
```